doc/configuration.rst

```diff
@@ -297,6 +304,51 @@ sender
   Default value: luigi-client@<server_name>
 
 
+[batch_notifier]
+----------------
+
+Parameters controlling the contents of batch notifications send from the
+scheduler
+
+email-interval
+  Number of minutes between e-mail sends. Making this larger results in
+  fewer, bigger e-mails.
+  Defaults to 60.
+
+batch-mode
+  Controls how tasks are grouped together in the e-mail. Can be set to
+  "family", "all", or "unbatched_params". If "family", all tasks of the
+  same family are treated identically. If "all", only identical tasks
+  are grouped together. If "unbatched_params", tasks are grouped
+  together if they are of the same family and all parameters other than
+  batch parameters are equal.
+  Defaults to "unbatched_params"
```

Comment:
I was trying to keep this short, but I guess it requires some examples. I'll update it to be more clear.